docs/content/opa-1.md

```diff
@@ -217,3 +219,22 @@ In OPA v1.0, the OPA server will bind to the `localhost` interface by default.
 
 The `--v1-compatible` flag on the `opa run` command makes OPA employ configuration defaults that will eventually be used in OPA v1.0. 
 Binding to the `localhost` interface is one such default.
+
+## Running OPA in 1.0 compatibility mode
+
+OPA can be run in 1.0 compatibility mode by using the `--v1-compatible` flag. When this mode is enabled, the current release of OPA will behave as OPA v1.0 will eventually behave by default when it comes to the changes described [here](#whats-changing-in-opa-10).
+
+The `--v1-compatible` flag is currently supported on the following commands:
+
+* `build`
+* `check`*
+* `fmt`*
+* `eval`
+* `test`
```

Comment:
Do you think it would be helpful to add a short note about what the `--v1-compatible` flag does in each command. For example, in `test` it will parse modules/bundles per `v1` syntax.